node/forest_libp2p/src/rpc/behaviour.rs

```diff
@@ -25,8 +26,11 @@ impl RPC {
 
     /// Send an RPCEvent to a peer specified by peer_id.
     pub fn send_rpc(&mut self, peer_id: PeerId, event: RPCEvent) {
-        self.events
-            .push(NetworkBehaviourAction::SendEvent { peer_id, event });
+        self.events.push(NetworkBehaviourAction::NotifyHandler {
+            peer_id,
+            event,
+            handler: NotifyHandler::Any,
```

Comment:
I got a little confused as to what this handler is when I attempted to upgrade last time... I was tempted to do th same: `Any`. Glad you figured it out.